### PR TITLE
Adding Cilium K8s Performance Dashboard to grafonnet

### PR DESCRIPTION
**Pull request by Manda-supraja26**
## Type of change

- [ ] Refactor
- [X] New feature
- [ ] Bug fix
- [X] Optimization
- [ ] Documentation Update

## Description
 Adding Cilium K8S Performance Dashboard to Grafonnet.

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [X] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please describe the System Under Test.
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: assets/cilium-k8s-perf/panels.libsonnet [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            + options.legend.withCalcs([])
+        
+
+
+
+            
+
+
+
+
```
Comment (jtaleric) [inline assets/cilium-k8s-perf/panels.libsonnet:59]: Curious the spacing here?

File: assets/cilium-k8s-perf/queries.libsonnet [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+           prometheus.withExpr('cilium_ip_addresses')
+            + prometheus.withFormat('time_series')
+            + prometheus.withIntervalFactor(2)
+            + prometheus.withLegendFormat('{{ pod }} - {{ family }}')
+            + prometheus.withDatasource('$Datasource') 
+    },
+
+    ciliumContainerCPU: {
+        query():
+            prometheus.withExpr('sum(irate(container_cpu_usage_seconds_total{container=~\"cilium.*\",container!=\"cilium-operator.*\",namespace!=\"\"}[$interval])) by (instance,pod,container,namespace,name,service) * 100')
```
Comment (jtaleric) [inline assets/cilium-k8s-perf/queries.libsonnet:26]: why all these expressions? just pull all the containers in `namespace=cilium`

File: assets/cilium-k8s-perf/queries.libsonnet [hunk 3]
```diff
@@ -0,0 +1,364 @@
...
+            + prometheus.withFormat('time_series')
+            + prometheus.withIntervalFactor(2)
+            + prometheus.withLegendFormat('{{ instance }} - {{ pod }}')
+            + prometheus.withDatasource('$Datasource') 
+    },
+
+    ciliumConatinerMemory: {
+        query():
+            prometheus.withExpr('container_memory_rss{container=~\"cilium.*\",namespace!=\"\"}')
```
Comment (jtaleric) [inline assets/cilium-k8s-perf/queries.libsonnet:35]: Same comment as above, just look at the namespace

Comment (Manda-supraja26) [reply assets/cilium-k8s-perf/queries.libsonnet:35]: sure , i will do that.

File: assets/cilium-k8s-perf/panels.libsonnet (hunk 1 above)
Comment (Manda-supraja26) [reply assets/cilium-k8s-perf/panels.libsonnet:59]: i will update it .

**Comment by vishnuchalla** [comment]
Requesting a link to the previous and current dashboards to make sure they are exactly the same look and feel wise.

**Review by vishnuchalla — CHANGES_REQUESTED** [review]
Requesting a link to the previous and current dashboards to make sure they are exactly the same look and feel wise.

**Comment by Manda-supraja26** [comment]
@vishnuchalla 
old dashboard: http://dittybopper-dittybopper.apps.smanda-ocp1.perfscale.devcluster.openshift.com/d/V9F0pW2Sz/cilium-k8s-performance?orgId=1&refresh=30s

new dashboard: http://dittybopper-dittybopper.apps.smanda-ocp1.perfscale.devcluster.openshift.com/d/8750pWhSz/cilium-k8s-performance-dashboard?orgId=1&refresh=30s

**Pull request merged**